### DISCUSSION TITLE
Help with Troubleshooting Low Radio Static Sound via Headphones

### DISCUSSION THREAD

**Discussion by Sajad-Lx** [AAFOptimus/AAFFamilyDCHAudio#2] [Q&A]
Hello,

First of all, thank you for your work in making it possible to use these amazing audio mods. The application, sound settings, and installation process all worked perfectly.

However, I’ve encountered a minor issue. Through my headphones, I can hear a very faint radio static sound. It doesn’t increase with volume adjustments and completely stops when the volume is set to 0 or muted.

I apologize for any trouble, but if possible, could you guide me on how to troubleshoot this issue?

Specifications:

    OS: Windows 11 23H2 (Build 22631.4602)
    AAF Version: AAFFamilyDCHAudioPack_9757
    Installed Components: Realtek APO, Dolby Atmos for Gaming, DTS X + Main Processing
    Realtek HWIDs:
        HDAUDIO\FUNC_01&VEN_10EC&DEV_0255&SUBSYS_10251343&REV_1000
        HDAUDIO\FUNC_01&VEN_10EC&DEV_0255&SUBSYS_10251343

Thank you in advance for your time and help

**Comment by AAFOptimus** [comment]
Thanks for the feedback. Firstly, I ask you to try the latest file I just uploaded. Don't forget to uninstall the package already in use.

**Comment by Sajad-Lx** [comment]
Apologies for the trouble. I believe the radio static issue might be caused by my laptop itself. 
I came to this conclusion because even after completely uninstalling the sound software and using the default "High Definition Audio Device" driver from Microsoft, the static sound persisted. This leads me to think it's a hardware-related issue with my laptop.

I'm also using your latest release and found no issues—everything is working flawlessly. The static sound still persisted regardless.
Thank you for the latest release.
